apps/admin-gui/src/app/shared/components/dialogs/add-role-dialog/add-role-dialog.component.html

```diff
@@ -1,47 +1,63 @@
 <div class="{{theme}}">
-  <h1 mat-dialog-title>{{'DIALOGS.ADD_ROLE.TITLE' | translate}}</h1>
+  <div mat-dialog-title>
+    <h1 mat-dialog-title>{{'DIALOGS.ADD_ROLE.TITLE' | translate}}</h1>
+    <perun-web-apps-immediate-filter
+      *ngIf="selectedRule?.primaryObject"
+      (filter)="filterValue = $event"
```

Comment:
When a filter value is typed in, but then the user swaps to a role that does not have the filter window and back to a role that does, the filter is empty, but the previous value is still used to filter the objects